tests/lib/log/log.go

```diff
@@ -0,0 +1,59 @@
+package log
```

Comment:
I definitely dislike this logger. It has a wrong logger interface  *and* couple the logger with the output destination. It's a file-logger.

Why not create an API that's just a wrapper to std `log` package passing the  io.Writer of the log file created (if opt in) ? 

If you don't like to use Go 'log' package, then at least use the same printer API (eg.: log.Printf), it's very useful.

In case of the CI tests, the logger output **must** be stdout, then I believe the logger interface must have a way to set their io.Writer to /dev/stdout. This is required to solve our current CI tests not working...

This logger adding a command-line argument to tests, like `go test -logger=stdout` would be nice also.